packages/shared_preferences/shared_preferences_foundation/example/ios/Runner/Assets.xcassets/AppIcon.appiconset/Contents.json

```diff
@@ -107,6 +107,12 @@
       "idiom" : "ipad",
```

Comment:
I know we've talked about this, but do we usually check in the Flutter app icons to example apps in the plugins repo (I can't comment on the binaries themselves)?

Reply:
Currently we do; I filed https://github.com/flutter/flutter/issues/116614 after earlier discussion but haven't implemented anything yet.